src/audio/output.rs

```diff
@@ -26,8 +26,7 @@ impl Output {
     /// Play a sound once.  A volume of 1.0 is unchanged, while 0.0 is silent.
     ///
     /// This will return an Error if the loaded audio file in source could not be decoded.
-    pub fn try_play_once(&self, source: &Source, volume: f32)
-        -> Result<(), DecoderError> {
+    pub fn try_play_once(&self, source: &Source, volume: f32) -> Result<(), DecoderError> {
```

Comment:
Its weird how sometimes it wants it on one line, and sometimes not.

Reply:
Yup.